### DISCUSSION TITLE
Remark MDX components

### DISCUSSION THREAD

**Discussion by gaurishhs** [remarkjs/remark#1096] [Q&A] [answered]
Hello! I'm using Vite, MDX and remark to transform `.md` and `.mdx` files and render then via React. I have a MDXProvider in my app and i pass my custom components to it, It works in MDX files but in markdown files code inside backticks does not get wrapped by my custom component. Please help me out

**Comment by wooorm** [comment]
Please provide much more info

**Reply by gaurishhs** [reply]
What more do you need?

**Reply by wooorm** [reply]
https://github.com/remarkjs/.github/blob/main/support.md#asking-quality-questions

**Comment by gaurishhs** [comment, marked as answer]
Basically i want to replace all codeblocks with a react component 

ANSWER:
I had not specified the `providerImportSource`, Sorry for bothering.

**Reply by wooorm** [reply]
And it does that

**Reply by gaurishhs** [reply]
no.  Imma get back to my pc and share some code

**Reply by gaurishhs** [reply]
I think i found the issue i had not specified the `providerImportSource`, Sorry for bothering.
